environments/production/terraform.yml

```diff
@@ -677,3 +677,11 @@ efs_file_systems:
     create_record: yes
     domain_name: "production.commcare.local"
     route_names: "shared-efs"
+
+terraform_imports:
+  - to: 'module.logshipping.aws_cloudwatch_event_rule.config-changes'
```

Comment:
You don't have quotes around this in the india/terraform.yml. Do you need it here?